apps/dashboard/src/pages/workflows.tsx

```diff
@@ -45,7 +45,7 @@ export const WorkflowsPage = () => {
   const navigate = useNavigate();
   const [searchParams, setSearchParams] = useSearchParams({
     orderDirection: DirectionEnum.DESC,
-    orderBy: 'updatedAt',
+    orderBy: 'createdAt',
```

Comment:
The thing is that we don't show the `createdAt` column, so this might result in a situation where users won't be able to sort back by the `createdAt`.
cc @SokratisVidros

Reply:
so we should probably introduce the `createdAt` column

Reply:
I don't think it's necessary. Sorting should be deterministic for now. We should be good.